src/main/java/io/prometheus/cloudwatch/WebServer.java

```diff
@@ -9,40 +14,80 @@
 import io.prometheus.client.exporter.MetricsServlet;
 
 public class WebServer {
+    private static final Logger LOGGER = Logger.getLogger(WebServer.class.getPackage().getName());
 
     public static String configFilePath;
+    public static int port;
 
-    public static void main(String[] args) throws Exception {
-        if (args.length < 2) {
-            System.err.println("Usage: WebServer <port> <yml configuration file>");
-            System.exit(1);
+    private static void enableDebugLogs() {
+      LOGGER.setLevel(Level.FINEST);
+      ConsoleHandler handler = new ConsoleHandler();
+      LOGGER.addHandler(handler);
+      for (Handler h: LOGGER.getHandlers()) {
+        h.setLevel(Level.FINEST);
+      }
+      LOGGER.log(Level.FINEST, "debug logging is on");
+    }
+
+    private static void printUsage() {
+        System.out.println("cloudwatch_exporter [-d] <port> <yml configuration file>");
+        System.out.println("-d,--debug     enable debug logs");
+    }
+
+    private static void parseOptions(String[] args) {
```

Comment:
No need for all this, the user can create a logging.properties.

Reply:
It's actually kind of nice to be able to pass this in as an option when you're running in a container. Then you don't have to worry about mounting another config file somewhere.

Reply:
My comment stands, there's no need to slowly reinvent how logging configuration is done in Java. Please remove this.